src/components/UI/SideMessages.css

```diff
@@ -0,0 +1,40 @@
+.message-group {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.center-message {
+  flex-grow: 4;
```

Comment:
question on these messages, Are they meant to grow and shrink? Your left and right messages appear to be fixed in width. Did you want them to have an initial width (kind of like a min-width) and grow from there if there is extra space?